ynput/OpenPype
# Ayon: deadline implementation

## Changelog Description
Quick implementation of deadline in Ayon. New Ayon plugin added for Deadline repository

## Additional info
Both implementation of OP and Ayon need to live together for a while, updated `GlobalJobPreLoad.py` to contain both. Simplest way chosen for first iteration.

!!! Known issue: Ayon Anatomy template don't contain `render` template as OP does and they should, imho.
Workaround it in for now in `submit_publish_job`, but still an issue in publish job in `Integrate`


## Testing notes:
1. deploy `modules/deadline/repository/custom` to DL repo
2. make sure that you have existing template name used in `tools/publish/template_name_profiles`
![AnyDeskMSI_wbCWA1kmc4](https://github.com/ynput/OpenPype/assets/4457962/b8eb4c65-c8f4-46e0-9217-03c7f5511b76)
(I used `publish` as `publish_render` is not yet set in Anatomy, but will be in the future)
3. run ayon mode and try to publish via DL (I was running `only_AE_DL` bundle in staging on ayon.dev)
4. run OP mode and try to publish via DL
